SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -79,22 +247,65 @@ void ASpatialDebugger::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& Out
 	DOREPLIFETIME_CONDITION(ASpatialDebugger, WorkerRegions, COND_SimulatedOnly);
 }
 
-void ASpatialDebugger::Tick(float DeltaSeconds)
+void ASpatialDebugger::BeginPlay()
 {
-	Super::Tick(DeltaSeconds);
+	Super::BeginPlay();
 
 	check(NetDriver != nullptr);
 
+	if (ensureMsgf(!NetDriver->SpatialDebuggerSystem.IsValid(), TEXT("SpatialDebugger system is already created at BeginPlay")))
+	{
+		const FSubView& DebuggerSubView = NetDriver->Connection->GetCoordinator().CreateSubView(
+			NetDriver->IsServer() ? SpatialConstants::ACTOR_AUTH_TAG_COMPONENT_ID : SpatialConstants::ACTOR_NON_AUTH_TAG_COMPONENT_ID,
+			[](const Worker_EntityId, const EntityViewElement& Element) -> bool {
+				return Element.Components.ContainsByPredicate(ComponentIdEquality{ SpatialConstants::SPATIAL_DEBUGGING_COMPONENT_ID });
```

Comment:
Should this filter also include an `IsClient()` somewhere? Isn't it correct for the debugging system on servers to see the entity before it has a debugging component?